docs/contributing.rst

```diff
@@ -44,6 +44,6 @@ Publishing
 
 .. code-block:: shell
 
-  $ python3 setup.py sdist bdist_wheel
-  $ twine check dist/*
-  $ twine upload dist/*
+  $ uv build
+  $ uv run twine check dist/*
+  $ uv run twine upload dist/*
```

Comment:
uv also has some upload capabilities: `uv publish`... https://docs.astral.sh/uv/guides/package/#publishing-your-package

Reply:
That's good to know, might be worth switching to that, but I'd want to look into it more.